### PR TITLE
add conqueror protector and dominator timeworn effects

### PR DESCRIPTION
**Pull request by Adamrch**

### CODE REVIEW

File: sim/core/stats/stats.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ const (
 	HealingPower
 	SpellDamage
 	FeralAttackPower
+	Timeworn
```
Comment (FelixPflaum) [inline sim/core/stats/stats.go:65]: Don't add a stat for this. See the note below that explicitly says not to add stats :D 

If at all make a pseudo stat for it, or just a stand-alone function that counts valid equipped items and use that inside the aura handlers for the rings.

File: sim/common/sod/item_effects/phase_6.go [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	SignetRingSubjugatorR5  = 234436
+	SignetRingSubjugatorR4  = 234437
+	SignetRingSubjugatorR3  = 234438
+	SignetRingSubjugatorR2  = 234439
+	SignetRingSubjugatorR1  = 234440
+	SignetRingConquerorR5   = 234202
+	SignetRingConquerorR4   = 234201
+	SignetRingConquerorR3   = 234200
+	SignetRingConquerorR2   = 234199
+	SignetRingConquerorR1   = 234198
```
Comment (kayla-glick) [inline sim/common/sod/item_effects/phase_6.go:45]: Normally I like to keep these ordered by ID but I think this makes a lot of sense like this here 👍 keeps it much more readable

**Comment by kayla-glick** [comment]
Resolves https://github.com/wowsims/sod/issues/1139

File: sim/core/stats/stats.go (hunk 1 above)
Comment (kayla-glick) [reply sim/core/stats/stats.go]: Converted to a pseudostat

**Pull request merged**